src/bnb_trading/fibonacci.py

```
@@ -165,76 +166,80 @@ def __init__(self, config: Dict[str, Any]) -> None:
         logger.info("Fibonacci анализатор инициализиран")
         logger.info(f"Ключови нива: {self.key_levels}")
         logger.info(f"Минимален swing размер: {self.min_swing_size:.1%}")
-    
+
     def find_swing_points(self, df: pd.DataFrame) -> Tuple[float, float, int, int]:
         """
         Намира последния swing high и swing low
-        
+
         Args:
             df: DataFrame с OHLCV данни
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)